config/application.rb

```diff
@@ -22,5 +22,17 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+    config.generators do |g|
+      g.test_framework :rspec,
+                       fixture: true,
+                       view_specs: false,
+                       helper_specs: false,
+                       routing_specs: false,
+                       request_specs: false,
+                       controller_specs: true
+
+      g.fixture_replacement :factory_girl, dir: 'spec/factories'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.